<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel_pracownika</class>
 <widget class="QDialog" name="Panel_pracownika">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pracownik:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_imie_pracownika">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Imie pracownika</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wyloguj">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wyloguj</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>681</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Moje zadania</string>
    </attribute>
    <widget class="QListWidget" name="listWidget_toDo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>241</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prace do zrobienia</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_doOceny">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>60</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prace wys≈Çane do oceny</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_done">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>241</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prace zrobione</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Panel pracy</string>
    </attribute>
    <widget class="QLabel" name="label_test">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>label</zorder>
  <zorder>label_imie_pracownika</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButton_wyloguj</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
